integration/examples/custom-buildx/skaffold.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: skaffold/v2beta13
+kind: Config
+
+build:
+  # Note that building for multiple platforms requires pushing to a registry,
+  # as the Docker Daemon cannot load multi-platform images.
+  local:
+    push: true
+  artifacts:
+  - image: skaffold-examples-buildx
+    custom:
+      # This buildCommand depends on a `docker buildx` builder to have been created
+      # with the name `skaffold-builder`:
+      #   $ docker buildx create --name skaffold-builder --platform linux/amd64,linux/arm64
```

Comment:
do we also need to add the `docker buildx  use skaffold-builder` command. 
or add the `--use`  flag to `docker buildx create --name skaffold-builder --platform linux/amd64,linux/arm64` command.

Reply:
I have a slight preference to explicitly specifying the builder with `--builder` as the message if the user *hasn't* created a builder and continues to use the default docker builder is cryptic:

    error: multiple platforms feature is currently not supported for docker driver. Please switch to a different driver (eg. "docker buildx create --use")


Whereas explicitly specifying `--builder xxx` where `xxx` doesn't exist fails with a slightly more coherent message:

    error: open /Users/bdealwis/.docker/buildx/instances/foo: no such file or directory